fedimintd/src/main.rs

```diff
@@ -8,7 +8,7 @@ use fedimint_server::config::{load_from_file, ServerConfig};
 use fedimint_server::consensus::FedimintConsensus;
 use fedimint_server::ui::run_ui;
 use fedimint_server::FedimintServer;
-use fedimint_wallet::{bitcoincore_rpc, Wallet};
+use fedimint_wallet::Wallet;
```

Comment:
Could we somehow re-export the relevant parts from the wallet module? I think we should do that more in general.

Reply:
Possibly yes, but my end goal is to have module repos be imported only be end binaries to instantiate the module only to register them in the core. The more code we move out of modules the easier it becomes.